packages/waku/src/client.ts

```diff
@@ -50,7 +50,7 @@ export const fetchRSC = cache(
     const options = {
       async callServer(actionId: string, args: unknown[]) {
         const response = fetch(
-          basePath + encodeInput(encodeURIComponent(actionId)),
+          basePath + encodeURI(encodeInput(encodeURIComponent(actionId))),
```

Comment:
This seems too ad-hoc. `input` is used for both components and actions.
I was hoping `encodeURIComponent` works, but if it doesn't escape enough, maybe we should use base64.
We can of course re-consider `encodeInput` implementation, but encoding too much is bad in DX.